static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -95,14 +103,16 @@ require.def(
                     this._registerEventHandlers();
```

Comment:
Yes, I tried to declare variable seperately, but I still get the same error from unit tests.
Error originates from:
.\static\script\runtimecontext.js:58